### ISSUE TITLE
[Suggestion]  momentarily display progress during --skip-ok

### ISSUE THREAD

**Issue by satori80** [rhash/RHash#167]
I don't know how onerous it would be to implement this, but it would be convenient to have a key that would show what is being processed by rhash during large "--skip-ok -c <hashfile>" operations. I.e, press the space key and see a single line of output to get an idea of how far along it is.

**Comment by rhash** [comment]
Why the `--percents` option is not enough?

**Comment by satori80** [comment]
I was thinking of something more along the lines of Issue #181, but not a toggle on/off. Just an update of how far along it is overall at the moment of the key press. I thought that constantly calculating & printing to term seems like a waste of resources.

**Comment by rhash** [comment]
Actually CPU is heavily loaded by calculating a hash. Compared to this, progress calculation is instantaneous, and does not eat resources.

Progress printing is also optimized. RHash prints progress no more then 20 times per second, and only if the percent value have changed. That means:
* For small files RHash prints progress only twice per file (0% on start and 100% on finish).
* For big files RHash prints progress 101 time per file (0%, 1%, ... 100%).

To summarize, the printing time can also be neglected.

Also processing a key press is not a Unix-way solution, so such suggestion will not be supported.

**Comment by satori80** [comment]
I can think of any number of *nix programs that enter or exit modes of operation with a single or combo key press, and many more that parse "Q" to exit cleanly. In fact, a project "John the Ripper" implements pretty much exactly what I suggested here, but I'll accept your decision and say no more.
